login/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en" >
+
+<head>
+  <meta charset="UTF-8">
+  <title>Login - globuzzer</title>
+  
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:100" rel="stylesheet">  
+      <link rel="stylesheet" href="login.css">
+
+  
+</head>
+
+<body>
+
+  <div class="main">
+  <div class="bg">
+  </div>
+  
+    
+      
+  <div class="content">
+    <div class="button-container">
+      <button class="button" onclick="materialClick(event)"><div class="circle animate"></div><span class="sign-in">Get Started</span><div class="loader"></div></button>

```

Comment:
split things on new lines , every tag his line , and remove extra spaces in the file, there are a lot of empty lines :)